src/locales/de/translation.json

```diff
@@ -81,6 +81,7 @@
   "Close modal": "Modale Fenster schließen",
   "Debug": "Debug",
   "Options": "Optionen",
+  "<0>Open in OSM</0> &mdash; Opens the current view on openstreetmap.org": "<0>Öffnen in OSM</0> &mdash; Öffnet die aktuelle Ansicht auf openstreetmap.de",
```

Comment:
I believe this is not needed, looks like a bug in i18n.
Otherwise this looks great, thanks!